packages/playground/wordpress/src/boot.ts

```diff
@@ -256,3 +257,47 @@ async function installWordPress(php: PHP) {
 			})
 	);
 }
+
+export async function getLoadedWordPressVersion(
+	php: PHP,
+	wordpressRoot: string
+): Promise<string | undefined> {
+	const result = await php.run({
+		code: `<?php
+			if (is_file('${wordpressRoot}/wp-includes/version.php')) {
+				require '${wordpressRoot}/wp-includes/version.php';
+				echo $wp_version;
+			}
+		`,
+	});
+
+	if (result.errors.length > 0) {
+		// TODO: Find a way to warn
+		return undefined;
+	}
+
+	let loadedVersion = '';
+	const wpVersion = result.text;
+	if (wpVersion) {
+		if (wpVersion.includes('-alpha')) {
+			loadedVersion = 'nightly';
+		} else if (wpVersion.includes('-beta')) {
+			loadedVersion = 'beta';
+		} else {
+			const majorMinorMatch = wpVersion.match(/^(\d+\.\d+)\.\d+$/);
+			loadedVersion = majorMinorMatch ? majorMinorMatch[1] : '';
+		}
+	}
+
+	if (!loadedVersion) {
+		// TODO: Is there something better than throwing at this stage?
+		return undefined;
```

Comment:
If this is empty, something is very wrong – I'd rather show a big error.

Reply:
OK, I will make this throw. That seems preferable for clarity, especially if we are planning to remove the uncertainty around requesting remote assets from older point releases. In that case, we are no longer running OK by coincidence but because we're actually doing the right thing.